src/catalog/catalog.cpp

```diff
@@ -606,8 +602,22 @@ ResultType Catalog::DropIndex(oid_t index_oid,
   return ResultType::SUCCESS;
 }
 
+ResultType Catalog::DropIndex(const std::string &index_name,
+                              concurrency::TransactionContext *txn){
+    if(txn == nullptr){
+        throw CatalogException("Do not have transaction to drop index " + index_name);
+    }
+    auto index_object = catalog::IndexCatalog::GetInstance()->GetIndexObject(index_name,txn);
```

Comment:
Nit: There needs to be a space after every argument in the function call. Same change in line 614